__tests__/formats/all.test.js

```diff
@@ -39,11 +39,7 @@ describe('formats', () => {
   const globalDate = global.Date;
 
   beforeAll(() => {
-    global.Date = class extends Date {
```

Comment:
@dbanksdesign @chazzmoney @didoo I found a better way to do the constant date trick, in the previous way it was messing up when I added the ES6 support for JEST where it was kinda transforming this code and giving me a very cryptic error; But now it's fixed

Reply:
As long as it works as expected! Nice work!